### ISSUE TITLE
Edit text we fill into certificates [8, 3, 2.67]

### ISSUE THREAD

**Issue by rsgonzal** [Iridescent-CM/technovation-app#2917]
Changes need to be made to include the season

### QA Steps
1. Login as nick+8j@technovation.org (10 judge)
2. Login as kate+judge21@technovation.org	(6-9 judge)
3. Login as mackenzie@technovation.org	(5 judge)
4. Login as rebecca+mexicostudent@technovation.org (semi)
5. Login as mackenzie+1stestteambuild@technovation.org (participation)
5. Login as roxana+twotest@technovation.org (completion)
6. Login as roxana+mail2@technovation.org (mentor certs)

Verify that certificates look good, new templates and language that should include the year.

**Comment by rsgonzal** [comment]
Text edited here: https://docs.google.com/document/d/1UpqRnLtqef11zq4FvqEt9gG0U-n_jonMOyrpNA8Zj3E/edit?pli=1#heading=h.ahp1yur4rcdp

**Comment by viviancan** [comment]
Sample updated certificates

![Screen Shot 2021-05-26 at 3.56.02 PM.png](https://images.zenhubusercontent.com/606b0f4fd13d53dff3d14fa5/639fb550-3602-4a3f-9782-a36ebfc25b94)![Screen Shot 2021-05-26 at 4.00.33 PM.png](https://images.zenhubusercontent.com/606b0f4fd13d53dff3d14fa5/74b278e8-d842-490c-a3d4-2f0d1c0e7431)

**Comment by kate-technovation** [comment]
1. Logged in as kate+judge21@technovation.org (6-9 judge):
<img width="1109" alt="Screen Shot 2021-05-27 at 3 51 08 PM" src="https://user-images.githubusercontent.com/57409606/119888447-5cf06e80-bf03-11eb-82a5-65ac29c64096.png">

2. Verified that certificate looks good, new template and language includes the year:
<img width="1020" alt="Screen Shot 2021-05-27 at 3 51 27 PM" src="https://user-images.githubusercontent.com/57409606/119888481-67126d00-bf03-11eb-9c3a-0e00373b4881.png">

**Comment by ophelieh** [comment]
STUDENTS
* Student with completed submission - I didn't have access to the certificate https://technovation-qa.herokuapp.com/admin/participants/611#/
* Student with partial submission (50% or more)- _we need to create a team in that case! All teams are either submitted or under 50%_
* Student who did not submit- no score or certificate as expected 
* Student semifinalist - I didn't have access to the certificate - https://technovation-qa.herokuapp.com/admin/participants/658#/ 

MENTORS
* Mentor with completed submissions - couldn't find where to get my certificates - https://technovation-qa.herokuapp.com/admin/participants/597#/ 
* Mentor with partial submission (50% or more)- _we need to create a team in that case! All teams are either submitted or under 50%_
* Mentor who did not submit - no certificate or scores as expected. It was weird that I could click on my submission when it was 0% 
* Mentor semifinalist - couldn't find the place to get my certificate - https://technovation-qa.herokuapp.com/admin/participants/659#/ 

JUDGES
* Judge less than 5 submissions - no certificate as expected
* Judge 5 - got the certified judge certificate which is weird since I had 6 scores (+1 deleted) - https://technovation-qa.herokuapp.com/admin/participants/734#/ 
* Judge 6-9 - picked a mentor judge and was confused for a minute as it brought me to my mentor profile instead of my judge one. Once, I got to the judge page, I got the Head judge certificate and looked as expected 
* Judge 10+ - judge advisor certificate. Looks good!

**Comment by rsgonzal** [comment]
Hey @ophelieh! we didn't generate the certificates for everyone. I just did for the ones above so that you could check the templates/text. Going to include the accounts here to view
> STUDENTS
> 
> * Student with completed submission - I didn't have access to the certificate https://technovation-qa.herokuapp.com/admin/participants/611#/

https://technovation-qa.herokuapp.com/admin/participants/741#/

> * Student with partial submission (50% or more)- _we need to create a team in that case! All teams are either submitted or under 50%_

https://technovation-qa.herokuapp.com/admin/participants/714#/

> * Student who did not submit- no score or certificate as expected
> * Student semifinalist - I didn't have access to the certificate - https://technovation-qa.herokuapp.com/admin/participants/658#/

https://technovation-qa.herokuapp.com/admin/participants/614#/
> 
> MENTORS
> 
> * Mentor with completed submissions - couldn't find where to get my certificates - https://technovation-qa.herokuapp.com/admin/participants/597#/
> * Mentor with partial submission (50% or more)- _we need to create a team in that case! All teams are either submitted or under 50%_
> * Mentor who did not submit - no certificate or scores as expected. It was weird that I could click on my submission when it was 0%
> * Mentor semifinalist - couldn't find the place to get my certificate - https://technovation-qa.herokuapp.com/admin/participants/659#/
> 
> JUDGES
> 
> * Judge less than 5 submissions - no certificate as expected
> * Judge 5 - got the certified judge certificate which is weird since I had 6 scores (+1 deleted) - https://technovation-qa.herokuapp.com/admin/participants/734#/
> * Judge 6-9 - picked a mentor judge and was confused for a minute as it brought me to my mentor profile instead of my judge one. Once, I got to the judge page, I got the Head judge certificate and looked as expected
> * Judge 10+ - judge advisor certificate. Looks good!

**Comment by ophelieh** [comment]
Ohhh makes sense. I must have misread the instructions. 

STUDENTS

* Student with completed submission - https://technovation-qa.herokuapp.com/admin/participants/741#/. Looks good except for the symbols in the name

![Screen Shot 2021-06-11 at 10 31 55 AM](https://user-images.githubusercontent.com/32206421/121711456-3fdda300-caa0-11eb-8703-4f6d7c5a1e32.png)

* Student with partial submission (50% or more)- https://technovation-qa.herokuapp.com/admin/participants/714#/. It Shows 2 certificate buttons. Both bring me to the certificate. The certificate looks good. Maybe to keep consistent we just want to write "in the Technovation Girls 2021 season" like for the completion one. 

![Screen Shot 2021-06-11 at 10 33 46 AM](https://user-images.githubusercontent.com/32206421/121711698-8206e480-caa0-11eb-957c-4fea0a451527.png)

Student semifinalist - https://technovation-qa.herokuapp.com/admin/participants/614#/. Certificate looks good! 

For ease of reading, I would add a period after the country and team name for all the student certificates so it reads "as a member of Technovation Mexico, team Libre Fiesta, to develop..."

![Screen Shot 2021-06-11 at 10 37 55 AM](https://user-images.githubusercontent.com/32206421/121712388-51737a80-caa1-11eb-9492-73926b2af1bb.png)

MENTORS

Love that they have to do the survey first! I would remove the 2020 and just keep 2021 to read "during the 2021 Technovation Girls season". Actually, I would reword the whole thing. It's kinda hard to follow along. Could we change the text to: "In recognition of your excellence in mentoring [TEAM NAME] team during the 2021 Technovation Girl season."?

Thank you!

**Comment by mdancho** [comment]
Certified, Advisor and Head judge appreciation certificate language look good. There is an extra space in between the first and last name on the judge account I tested. Doesn't seem like there is a space on the platform so not sure why that is displaying this way.  
<img width="797" alt="Screen Shot 2021-06-14 at 10 25 44 AM" src="https://user-images.githubusercontent.com/52216992/121933729-49af1280-ccfb-11eb-9bfb-53dc503a7879.png">

For mentors, the certificate looks good. I am comfortable with the edit that Ophelie reccomended.

**Comment by rsgonzal** [comment]
thank you for testing!!

@ophelieh i tagged you in the google doc for the comma edits you requested for semifinalist & completion

The participation & mentor certificates are edited by audra. I think we are going to have to leave as is for this season. We can work on consistency for next!

For the judge certificate, I'm not sure why the space is large or if it just happens to be because of the letters. any thought, @viviancan ?

**Comment by viviancan** [comment]
@rsgonzal @mdancho Looks like there was a trailing space on the "kenzarooniie" first name that was saved in the DB.

I logged in as the participant and erased the trailing space and the certificate looks normal now!
![Screen Shot 2021-06-15 at 9.57.43 AM.png](https://images.zenhubusercontent.com/606b0f4fd13d53dff3d14fa5/df99c020-e70f-4580-906c-dba6b66da1b4)

**Comment by rsgonzal** [comment]
thanks for making the update, vivian!
